mmcomplex.raw.html

```diff
@@ -750,39 +748,32 @@
 <I>f</I>(3),... of all reals, thereby falsifying this claim.
 
 <P>Here is how we construct this number.  We construct, in parallel, two
-auxiliary sequences <I>g</I>(1), <I>g</I>(2), <I>g</I>(3),... and
-<I>h</I>(1), <I>h</I>(2), <I>h</I>(3),... derived from <I>f</I>(1),
+auxiliary sequences <I>g</I>(0), <I>g</I>(1), <I>g</I>(2),... and
```

Comment:
I think the original indices were right, see the definition https://us.metamath.org/mpeuni/df-nn.html

Reply:
`f` is indexed by `NN`, `g` and `h` are indexed by `NN0`.  Look at https://us.metamath.org/mpeuni/ruclem6.html and note especially 𝐹:ℕ⟶ℝ and 𝐺:ℕ0⟶(ℝ × ℝ)

Reply:
you're right